provider/cmd/pulumi-gen-eks/main.go

```diff
@@ -1682,9 +1682,14 @@ func vpcCniProperties(kubeconfig bool) map[string]schema.PropertySpec {
 		},
 		"image": {
 			TypeSpec: schema.TypeSpec{Type: "string"},
-			Description: "Specifies the container image to use in the AWS CNI cluster DaemonSet.\n\n" +
+			Description: "Specifies the aws-node container image to use in the AWS CNI cluster DaemonSet.\n\n" +
 				"Defaults to the official AWS CNI image in ECR.",
 		},
+		"nodeImage": {
```

Comment:
Would "nodeAgentImage" be a clearer name here? To me, `image` and `nodeImage` could be the same thing without reading the description.

Reply:
`nodeAgentImage` makes it clearer. Will update the PR. Thanks!